cloud/google/compute/resources/instancegroup.go

```diff
@@ -338,6 +349,26 @@ func (r *InstanceGroup) Delete(actual cloud.Resource, immutable *cluster.Cluster
 	return renderedCluster, actual, nil
 }
 
+const (
```

Comment:
Maybe we should push it to the of the page, with other constants, so it's not confusing